### PR TITLE
makes black terrors and white terrors better

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Black and white terrors now deal 30 stamina damage on bite
Black and white terrors can inject through hardsuits if you're in staminacrit

## Why It's Good For The Game
Neither black or white terrors are very good right now, and especially after modsuits get merged they'll suffer even more.
White terrors are the better of the two but just suffer from being too easy to lock down, glass cannons without much firepower to speak of.
Black terrors are a pick class that can't take picks, anyone you want to kill is wearing peirce immune shit.
This pr fixes both of those issues

## Testing
Skrell got a bit a bunch, followed by doing the same stuff with a hardsuit

## Changelog
:cl:
tweak: Black and white terrors now deal 30 stamina damage on bite
tweak: Black and white terrors can inject through hardsuits if you're in staminacrit
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm [hunk 1]
```diff
@@ -34,12 +34,13 @@
 	if(L.reagents.has_reagent("terror_black_toxin", 100))
 		return ..()
 	var/inject_target = pick("chest", "head")
-	if(L.IsStunned() || L.can_inject(null, FALSE, inject_target, FALSE))
+	L.adjustStaminaLoss(30)
+	L.attack_animal(src)
+	if(L.can_inject(null, FALSE, inject_target, FALSE) || HAS_TRAIT(L, TRAIT_HANDS_BLOCKED) && HAS_TRAIT(L, TRAIT_IMMOBILIZED))
```
Comment (Contrabang) [inline code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm:39]:
```suggestion
	if(L.can_inject(null, FALSE, inject_target, FALSE) || (HAS_TRAIT(L, TRAIT_HANDS_BLOCKED) && HAS_TRAIT(L, TRAIT_IMMOBILIZED)))
```

File: code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm [hunk 2]
```diff
@@ -40,7 +40,8 @@
 	var/inject_target = pick("chest","head")
 	L.attack_animal(src)
 	L.KnockDown(10 SECONDS)
-	if(L.IsStunned() || L.IsParalyzed() || L.can_inject(null, FALSE, inject_target, FALSE))
+	L.adjustStaminaLoss(30)
+	if(L.can_inject(null, FALSE, inject_target, FALSE) || (HAS_TRAIT(L, TRAIT_HANDS_BLOCKED) && HAS_TRAIT(L, TRAIT_IMMOBILIZED)))
```
Comment (Contrabang) [inline code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm:44]: ditto here

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm (hunk 2 above)
Comment (GDNgit) [reply code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm:44]: i did that there

File: code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm (hunk 1 above)
Comment (GDNgit) [reply code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm]: applied

File: code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm (hunk 2 above)
Comment (Contrabang) [reply code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm:44]: im blind

Comment (GDNgit) [reply code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm:44]: i'm blind too cuz i managed to only do it once in the code

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**